plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties

```diff
@@ -101,3 +93,8 @@ action.stop=Stop
 action.remove=Remove
 job.terminated.template=<terminated> {0}
 deploy.console.name=App Engine Deploy
+
+ok.button=OK
```

Comment:
Unnecessary if you use `MessageDialog.openError`.

Reply:
good, done